.github/workflows/distcheck.yaml

```diff
@@ -69,13 +68,6 @@ jobs:
             configmanager: dnf config-manager
             cudapackages: cuda-cudart-devel-12-6 cuda-crt-12-6
 
-          - container: public.ecr.aws/amazonlinux/amazonlinux:2
-            displayname: al2
-            efainstallerdir: ALINUX2
-            nvidiadistro: rhel7
-            configmanager: yum-config-manager
-            cudapackages: cuda-cudart-devel-12-4 cuda-crt-12-4
-
     runs-on: ubuntu-latest
```

Comment:
@rajachan pointed out that this might be avoidable if you hardcode ubuntu 20 here instead.

Reply:
Trying in #729

Reply:
No luck 😞